Makefile

```diff
@@ -107,7 +107,7 @@ generate: ## Code generation (requires operator-sdk >= v0.5.0)
 build: ## build binaries
 	@# version must be of the form v1.2.3 with optional suffixes -4 and/or -g56789ab
 	@# or the binary will crash when it tries to parse its version.Raw
-	@echo $(VERSION) | grep -qP '^v\d+\.\d+\.\d+(-\d+)?(-g[a-f0-9]{7,})?(\.p\d+)?(-dirty)?$$' || \
+	@echo $(VERSION) | grep -qP '^v\d+\.\d+\.\d+(-\d+)?(-g[a-f0-9]{7,})?(\.[a-zA-Z0-9_.-]+)?(-dirty)?$$' || \
```

Comment:
`\.` ?

Reply:
It didn't change behavior on my version of grep, but I trust you more than it :) . Updated.